src/components/landing/Landing.jsx

```diff
@@ -5,13 +5,17 @@ import ProjectTypes from "@/components/landing/ProjectType";
 import PastProjects from "@/components/landing/PastProjects";
 import ViewMore from "@/components/ViewMore";
 import Footer from "@/components/Footer";
+import Filterbuttons from "../projects/filterbuttons";
 
 const Landing = () => {
   return (
     <div className="w-full bg-blue-50 flex flex-col">
       <Navbar />
       <Welcome />
       <About />
+
+      <Filterbuttons />
```

Comment:
filter buttons shouldn't be in the landing page, it should be in the projects page